samples/nrf9160/modem_shell/src/gnss/gnss_interface.c

```diff
@@ -598,6 +740,103 @@ static void handle_timeout_work_fn(struct k_work *item)
 	}
 }
 
+#if defined(CONFIG_NRF_CLOUD_PGPS)
+static void get_pgps_data_work_fn(struct k_work *work)
+{
+	ARG_UNUSED(work);
+
+	int err;
+
+	mosh_print("GNSS: Getting P-GPS predictions from nRF Cloud...");
+
+	err = nrf_cloud_jwt_generate(0, jwt_buf, sizeof(jwt_buf));
+	if (err) {
+		mosh_error("GNSS: Failed to generate JWT, error: %d", err);
+		return;
+	}
+
+	struct nrf_cloud_rest_context rest_ctx = {
+		.connect_socket = -1,
+		.keep_alive = false,
+		.timeout_ms = CONFIG_NRF_CLOUD_REST_RECV_TIMEOUT * MSEC_PER_SEC,
+		.auth = jwt_buf,
+		.rx_buf = rx_buf,
+		.rx_buf_len = sizeof(rx_buf),
+		.fragment_size = 0, /* Defaults to CONFIG_NRF_CLOUD_REST_FRAGMENT_SIZE when 0 */
```

Comment:
Is CONFIG_NRF_CLOUD_REST_FRAGMENT_SIZE visiblein this file? If so, you can just write: .fragment_size = CONFIG_NRF_CLOUD_REST_FRAGMENT_SIZE, 

there is similar line for A-GPS also

Reply:
I guess it is, but then again there's functionality for this in the nRF Cloud library. :) I think we could just omit the parameters which are left zero or NULL.

Reply:
As discussed offline, I'll remove parameters which do not need to be initialized.